deploy/docker-compose.yml

```diff
@@ -1,63 +1,79 @@
 version: "3.9"
 
-# Expose port to access s-pipes-editor-rest directly
-x-access-db-server-development-port: &s-pipes-editor-rest-dev-port
-  ports:
-    - "127.0.0.1:${EDITOR_REST_PORT:-18115}:18115"
-
 services:
+  nginx:
+    image: nginx:latest
+    ports:
+      - "127.0.0.1:${INTERNAL_HOST_PORT:-1235}:80"
+    restart: unless-stopped
+    depends_on:
+      - s-pipes-engine
+      - s-pipes-editor-rest
+      - db-server
+      - s-pipes-editor-ui
+    environment:
+      NGINX_ENVSUBST_OUTPUT_DIR: "/etc/nginx"
+      APP_ORIGIN: "${PUBLIC_ORIGIN:-http://localhost:${INTERNAL_HOST_PORT:-1235}}"
+      SERVICE_URL: "${SPIPES_EDITOR_REST:-http://s-pipes-editor-rest:18115}"
```

Comment:
I do not understand role of `SPIPES_EDITOR_REST`, when do you want to use it. Why it is not sufficient to use `http://s-pipes-editor-rest:18115` in nginx directly.

Reply:
Moreover in nginx.conf you have many services so it does not makes sense to me call it SERVICE_URL but be more specific.

Reply:
@blcham
The motivation behind configuring the SPIPER_EDITOR_REST url  was to make it easier to change the API for the frontend, if needed in the future. I can remove it and simply hardcode http://s-pipes-editor-rest:18115 if preferred.

You are also right about SERVICE_URL in nginx.conf—it should be updated to SPIPER_EDITOR_REST_URL to be more speicific.

Reply:
OK, I saw in README:
```
SPIPES_EDITOR_REST -- to set up different s-pipes-editor backend (defaults to internal docker service s-pipes-editor-rest:18115)
```

Thus, I suggest renaming it **only here** `SPIPES_EDITOR_REST_SERVICE_URL` as in the context of nginx.

Reply:
@palagdan